napari/layers/labels/labels.py

```diff
@@ -463,14 +468,42 @@ def seed(self):
 
     @seed.setter
     def seed(self, seed):
+        warnings.warn(
+            "seed is deprecated since 0.4.19 and will be removed in 0.5.0, please use seed_rng instead",
+            FutureWarning,
+            stacklevel=2,
+        )
+
         self._seed = seed
-        self.colormap.seed = seed
+        self.colormap = label_colormap(self.num_colors, self.seed)
         self._cached_labels = None  # invalidate the cached color mapping
         self._selected_color = self.get_color(self.selected_label)
         self.events.colormap()  # Will update the LabelVispyColormap shader
         self.refresh()
         self.events.selected_label()
```

Comment:
Is there a reason to keep this around instead of simply rerouting this to `seed_rng.setter`?

Reply:
because `seed_rng` is `int` not float.